content/robots/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Robots
+weight: 4
+---
+
+In this year's competition we have 2 Robots, which are each controlled by their own python file.
+
+- [Crane Robot](crane) which is controlled by `crane.py`.
+- [Forklift Robot](forklift) which is controlled by `forklift.py`.
```

Comment:
Thoughts on how we document what exists on the physical robot?

Reply:
Ah I didn't know students would be programming the physical robot, I guess it makes sense to add a new page to `robots/`, maybe in a separate PR